de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/LocalEditorHandler.java

```diff
@@ -242,8 +236,9 @@ public void saveDocument(
      * @param file
      */
     public void activateEditor(@NotNull VirtualFile file) {
-        SPath path = virtualFileConverter.convertToPath(file);
-        if (path != null) {
+        SPath path = VirtualFileConverter.convertToSPath(file);
+
+        if (path != null && SessionUtils.isShared(path)) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "SessionUtils"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=294&issues=AWcxYM7E0EcSutaxLwSd&open=AWcxYM7E0EcSutaxLwSd)

[ISSUE_KEY]: AWcxYM7E0EcSutaxLwSd